ultravox/tools/ds_tool/user_messages.jinja

```diff
@@ -0,0 +1,7 @@
+[
```

Comment:
Should you still have the square braces if you're using YAML?

Reply:
Hmm, I think it's possible to do it without the braces, but then I'll have to be really careful about number of tabs/newlines which is not easy to figure out in Jinja.

With the braces it's way more forgiving on those types of things.